tests/test_validate.py

```diff
@@ -12,7 +12,7 @@ def test_valid_bundles():
         with open(bundle) as f:
             bundle = yaml.safe_load(f)
             bundle = unformat_bundle(bundle)
-            valid = ValidateCmd().validate(bundle)
+            valid, json = ValidateCmd().validate(bundle)
```

Comment:
Hmm, I realized that what ValidateCmd returns is not json (string) but python dict, so it should be named differently.
Maybe `results`

Reply:
Good idea

Reply:
Addressed in the latest commit.